### PR TITLE
Add prometheus application metrics faq entry

### PR DESCRIPTION
**Pull request by douglas-reid**

### CODE REVIEW

File: content/help/faq/telemetry/prometheus-application-metrics.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- `prometheus.io/scrape: "true"`
+- `prometheus.io/path: "<metrics path>"`
+- `prometheus.io/port: "<metrics port>"`
+
+A few notes:
+
+- If the Prometheus pod started before the Istio Citadel pod could generate the required certs and distribute them to Prometheus, the Prometheus pod will need to
+be restarted in order to collect from mTLS-protected targets.
+- In mTLS-enabled environments, you will need to add the Prometheus metrics port to the service and deployment specifications.
```
Comment (mandarjog) [inline content/help/faq/telemetry/prometheus-application-metrics.md]: If your application exposes prometheus metrics on a dedicated port, that port should be added to the service and deployment specifications.

**Comment by mandarjog** [comment]
@geeknoid  please review / approve.

**Comment by geeknoid** [comment]
Also, please rebase in order to address the link-related CI failures.

File: content/help/faq/telemetry/prometheus-application-metrics.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Can I use Prometheus to scrape application metrics with Istio?
+weight: 90
+---
+
+Yes. Istio ships with [configuration for Prometheus](https://raw.githubusercontent.com/istio/istio/release-1.1/install/kubernetes/helm/subcharts/prometheus/templates/configmap.yaml)
+that enables collection of application metrics in both mTLS-enabled and non-mTLS environments.
+
+The `kubernetes-pods` job collects application metrics from pods in non-mTLS protected environments. The `kubernetes-pods-istio-secure` job collects metrics
```
[the remark this answers is not in the document]
Comment (douglas-reid) [reply content/help/faq/telemetry/prometheus-application-metrics.md]: Done here and elsewhere.

File: content/help/faq/telemetry/prometheus-application-metrics.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Can I use Prometheus to scrape application metrics with Istio?
+weight: 90
+---
+
+Yes. Istio ships with [configuration for Prometheus](https://raw.githubusercontent.com/istio/istio/release-1.1/install/kubernetes/helm/subcharts/prometheus/templates/configmap.yaml)
+that enables collection of application metrics in both mTLS-enabled and non-mTLS environments.
```
[the remark this answers is not in the document]
Comment (douglas-reid) [reply content/help/faq/telemetry/prometheus-application-metrics.md]: done.

File: content/help/faq/telemetry/prometheus-application-metrics.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Can I use Prometheus to scrape application metrics with Istio?
+weight: 90
+---
+
+Yes. Istio ships with [configuration for Prometheus](https://raw.githubusercontent.com/istio/istio/release-1.1/install/kubernetes/helm/subcharts/prometheus/templates/configmap.yaml)
```
[the remark this answers is not in the document]
Comment (douglas-reid) [reply content/help/faq/telemetry/prometheus-application-metrics.md]: done.

**Comment by douglas-reid** [comment]
@geeknoid PTAL. I've (finally) rebased and converted `mTLS` to `mutual TLS`, etc.

**Pull request merged**